bundles/automation/org.eclipse.smarthome.automation.module.core.test/ESH-INF/automation/Xrules/ItemRuleDefinition.json

```diff
@@ -8,7 +8,7 @@
     ],
     "configuration": {},
     "description": "Sample Rule for items definition.",
-    "on": [  
+    "triggers": [
```

Comment:
This file is used in Eclipse Smart Home Automation Integration Test and the test was failing before fix file